content/applications/hr/payroll/configuration.rst

```diff
@@ -67,17 +67,9 @@ form:
 - :guilabel:`Code`: This code appears with the work entry type on timesheets and payslips. Since the
   code is used in conjunction with the *Accounting* application, it is advised to check with the
   accounting department for a code to use.
-- :guilabel:`Sequence`: The sequence determines the order that the work entry is computed in the
-  payslip list.
-- Check boxes: If any of the items in the list applies to the work entry, check off the box by
-  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
-  This field has a drop-down menu to select the specific type of time off, or a new type of time off
-  can be entered.
-
-.. image:: configuration/new-work-entry.png
-   :align: center
-   :alt: New work entry type form.
-
+- :guilabel:`DMFA Code`: This code is used to identify DMFA entries on a corresponding DMFA report,
```

Comment:
I am assuming that if a company is using this code they would have a better understanding of what DMFA is/refers to, so it probably doesn't need to be defined further, right?

Reply:
Correct- it's Belgian-only (i put that in the doc) so I don't think so. It sounds close to a social security number- we don't have to define that for us US people, so I figured the same for the Belgians =). Thanks for the review BTW!